src/main/java/edu/cornell/kfs/module/purap/rest/jsonObjects/PaymentRequestDto.java

```diff
@@ -0,0 +1,191 @@
+package edu.cornell.kfs.module.purap.rest.jsonObjects;
+
+import java.sql.Date;
```

Comment:
Is the use of java.sql.Date preferred due to our current codebase and our ORM?

Reply:
PurchaseOrderDocument uses java.sql.Date, so I thought using that Object in these DTOs would simplify the creation of the PO document in the next step

Reply:
Only commenting due to the fact that I was the developer that worked on a number of the date-time conversion JIRAs.  
IMHO, I feel that we need to use the standard/more current java.time classes and only utilize the java.sql.Date class as close where the actual database call happens.  java.sql.Date has fundamental flaws in the area of time zones that I do not think we want to introduce in this new code.